web/src/styles/components/errors.scss

```diff
@@ -1,16 +1,16 @@
 .input-error-bar {
-  border-left-width: 4px;
-  border-left-color: #ffffff;
-  border-left-style: solid;
-  left: -17px !important;
+  left: -13px !important;
   margin-left: 0 !important;
   padding-left: 13px !important;
   margin-right: -17px !important;
   position: relative;
 }
 
 .input-error-bar.error {
+  border-left-width: 4px;
   border-left-color: #d54309;
+  border-left-style: solid;
+  left: -17px !important;
   margin-bottom: 1em;
```

Comment:
We should be using `rem` in place of `px` for scalability/accessibility reasons.